pkg/plugins/policies/meshmetric/plugin/v1alpha1/plugin.go

```diff
@@ -113,10 +122,42 @@ func configurePrometheus(rs *core_xds.ResourceSet, proxy *core_xds.Proxy, promet
 	return nil
 }
 
+func configureOpenTelemetry(rs *core_xds.ResourceSet, proxy *core_xds.Proxy, openTelemetryBackend *api.OpenTelemetryBackend) error {
```

Comment:
Could we add a test here so we can see how the configuration is supposed to look?

Reply:
of course, I just started adding test for all new components

Reply:
I've added test